.github/workflows/CI.yml

```diff
@@ -84,6 +84,21 @@ jobs:
       run: |
         CHPL_HOME=$PWD make run-dyno-linters
 
+  make_check_llvm_none:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/${{ github.repository_owner }}/chapel-github-ci:latest
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.github_token }}
+    steps:
+    - uses: actions/checkout@v3
+    - name: make check_llvm_none
+      run: |
+       CHPL_LLVM=none ./util/printchplenv --all make make check
+       CHPL_HOME=$PWD ./util/test/run-in-test-venv.bash ./examples/hello.chpl
```

Comment:
I don't think it hurts to have this, but is there a specific reason we test `hello.chpl` in the `CHPL_LLVM=none` configuration but not the usual make check configuration?